README.md

```diff
@@ -1,201 +1,12 @@
 # News challenge
 
-* Feel free to use Google, your notes, books, etc. but work on your own.
-* If you refer to the solution of another coach or student, please put a link to that in
-  your README.
-* If you have a partial solution, **still check in a partial solution** and send in a pull
-  request.
-* You must submit a pull request to this repo with your code by 10am Monday morning.
+1. Download project files 
+2. Run 'npm install'
+3. Run npm run build 
+4. Into terminal, enter 'open html.com' 
```

Comment:
do you mean 'open index.html'? I think it's supposed to run from index.js but I'm not 100% clear on the practical difference